app-libs/stf/src/stf_sgx.rs

```diff
@@ -240,6 +247,43 @@ where
 	}
 }
 
+impl<Call, Getter, State, Runtime> EncointerSchedulerPalletInterface<State>
+	for Stf<Call, Getter, State, Runtime>
+where
+	State: SgxExternalitiesTrait,
+	Runtime: frame_system::Config
```

Comment:
This is most likely already enforced by other pallets configs, so you can remove the `frame_system::Config`, it's just boilerplate code.